workers/unity/Packages/io.improbable.gdk.buildsystem/WorkerBuilder.cs

```diff
@@ -112,14 +75,67 @@ public static void Build()
             }
         }
 
+        private static List<BuildContext> GetBuildContexts(IEnumerable<string> wantedWorkerTypes,
+            BuildEnvironment buildEnvironment, ScriptingImplementation? scriptImplementation = null,
+            ICollection<BuildTarget> buildTargetFilter = null)
+        {
+            var spatialOSBuildConfiguration = BuildConfig.GetInstance();
```

Comment:
```suggestion
            var spatialOsBuildConfiguration = BuildConfig.GetInstance();
```

We can still change our style guides, there is time!